.travis.yml

```diff
@@ -7,8 +7,12 @@ php:
   - nightly
   - hhvm
 
-install:
+before_script:
   - composer self-update
-  - composer install --prefer-source
+  - composer install --dev --prefer-source
+  - mkdir -p build/logs
```

Comment:
No, if you look the build failed, and double PHP unit call was what I thought the issue was. Strangely it builds without issue in my repo, so not sure what is wrong